migrator/migrations/m_103_to_m_104_networkpolicy_guidance/policies_before_and_after/after/deployment_has_ingress_network_policy.json

```
@@ -0,0 +1,149 @@
+{
+  "id": "38bf79e7-48bf-4ab1-b72f-38e8ad8b4ec3",
+  "name": "Deployments should have at least one ingress Network Policy",
+  "description": "Alerts if deployments are missing an ingress Network Policy",
+  "rationale": "Pods that lack ingress Network Policies have unrestricted reachability on the network and may be exposed to attack",
```

How about using plural here? Or adding `an` before the `attack` word if we keep it singular?

```suggestion
  "rationale": "Pods that lack ingress Network Policies have unrestricted reachability on the network and may be exposed to attacks",
```

---

That's probably a typo yea, I copied from Chris' message in [ROX-11110](https://issues.redhat.com/browse/ROX-11110).